src/utils/accounts.ts

```diff
@@ -7,8 +7,7 @@ import { BuidlerRuntimeEnvironment } from '@nomiclabs/buidler/types'
 export async function getRootAccount(
   bre: BuidlerRuntimeEnvironment
 ): Promise<string> {
-  const accounts =
-    bre.config.networks[bre.network.name].from ||
-    (await bre.web3.eth.getAccounts())
-  return accounts[0]
```

Comment:
Ouch haha, what a mistake!

I remember we changed this briefly before (for 0.2.5?) to consolidate all the requests for the root account into this function?

Reply:
Yeah, it was a regression bug myself introduced with that change 😔

Reply:
How didn't Typescript caught this? :thinking: Is `bre.web3.eth.getAccounts()` strongly typed?